event_dispatcher/before_after_filters.rst

```diff
@@ -133,16 +133,13 @@ event subscribers, you can learn more about them at :doc:`/event_dispatcher`::
         {
             $controller = $event->getController();
 
-            /*
-             * $controller passed can be either a class or a Closure.
-             * This is not usual in Symfony but it may happen.
-             * If it is a class, it comes in array format
-             */
-            if (!is_array($controller)) {
-                return;
+            // when a controller class defines multiple action methods, the controller
+            // is returned as [$controllerInstance, 'methodName']
```

Comment:
I'm not sure if this is completely true as in theory you can define `__invoke()` and `foo()` as actions. The invoke will cause the FQCN to be used as controller name, while the other would be `FQCN::foo`. It's more of a nitpick on the details and I don't see this scenario happen often.

Here's an example of the invokable controller that's also registered as service:
![image](https://user-images.githubusercontent.com/1754678/65329612-1c324500-dbb9-11e9-8bbc-0a6808cabdd3.png)

If the controller is a callable object or object with method, it will be an object in `$controller` or `$controller[0]` accordingly. In case of string controllers (function reference), it won't pass the instanceof check, so that's all covered here.